GenericCompiler.cmake

```diff
@@ -2,5 +2,6 @@ set (GEOS_Fortran_FLAGS_DEBUG   "${GEOS_Fortran_Debug_Flags} ${common_Fortran_fl
 set (GEOS_Fortran_FLAGS_RELEASE "${GEOS_Fortran_Release_Flags} ${common_Fortran_flags} ${GEOS_Fortran_Release_FPE_Flags} ${ALIGNCOM}")
 set (GEOS_Fortran_FLAGS_VECT    "${GEOS_Fortran_Vect_Flags} ${common_Fortran_flags} ${GEOS_Fortran_Vect_FPE_Flags} ${ALIGNCOM}")
 
-set (CMAKE_Fortran_FLAGS_DEBUG   "${GEOS_Fortran_FLAGS_DEBUG}" CACHE STRING "Debug Fortran flags" FORCE )
-set (CMAKE_Fortran_FLAGS_RELEASE "${GEOS_Fortran_FLAGS_RELEASE}" CACHE STRING "Release Fortran flags" FORCE )
+# Use separate_arguments to split options into semicolon separated list
+separate_arguments(GEOS_Fortran_FLAGS_RELEASE UNIX_COMMAND ${GEOS_Fortran_FLAGS_RELEASE})
```

Comment:
Is there a reason you are using `UNIX_COMMAND` instead of `NATIVE_COMMAND`? I know we don't support Windows, but it's nice to dream.

Reply:
Thanks, I didn't see `NATIVE_COMMAND`. I think that would be better.

Reply:
Fixed in cca5310e74780f5efea8988cc8c5d6ed436a354c